sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -325,6 +325,25 @@ def test_binomial():
 
     assert binomial(a, b).is_nonnegative is True
 
+    # issue #13980 and #13981
+    assert binomial(-7, -5) == 0
+    assert binomial(-23, -12) == 0
+    assert binomial(13/2, -10) == 0
+
+    assert binomial(-49, -51) == binomial(-49, -51)
```

Comment:
This test doesn't seem useful; an expression is equal to itself.

Reply:
As I mentioned below, the cases that do not get into any of the conditions, return as it is, meaning can be solved by rewriting. This particular case when rewritten yields `nan`. I guess rewriting needs to be improved

```
>>> binomial(-49, -51).rewrite(gamma)
nan
```

Reply:
But this test doesn't actually test that it remains unevaluated. Whatever the lhs gives, it will be the same on the rhs. If you want to test that it is unevaluated use `isinstance(binomial(-49, -51), binomial)`.